src/main/groovy/pl/allegro/tech/build/axion/release/domain/Releaser.groovy

```diff
@@ -1,9 +1,11 @@
 package pl.allegro.tech.build.axion.release.domain
 
 import com.github.zafarkhaja.semver.Version
+import org.eclipse.jgit.api.errors.GitAPIException
```

Comment:
releaser is part of domain, while `jgit` is an implementation detail - this exception should be wrapped into some kind of domain exception (like `ScmException`). `jgit` should be only visible within `GitRepository`

Reply:
That's so very true, silly of me to have missed this leak.